Applications/WebEditor.qmd

```diff
@@ -63,48 +63,96 @@ Based on their applications user can build their model by simple drag and drop m
 
 1. Load Collection 
    
-    In order to load the required collection make sure it is available by searching in the sidebar. Once you find your collection you can simply drag and drop it for carrying out further actions. In the following clip, you can observe the sequential actions are taken to accomplish the following tasks: choosing a collection of interest, defining the spatial and temporal boundaries, and filtering the necessary bands for subsequent processing. Specifically, for calculating the NDVI, the Red band (B04) and Near-Infrared (NIR) band (B08) have been selected.
+    a. To start a task within the openEO web editor, your first step involves ensuring you have the necessary dataset for the analysis. Thus, you must ensure the required dataset exists within the openEO database. You can do this by exploring/searching the list of available datasets from the sidebar of the interface.
+    
+    b. In our example, we want to calculate the Normalized Difference Vegetation Index (NDVI), so the Sentinel 2 L2A collection will be used. Once you find this collection, the next step is to drag and drop it into the Process Editor for further operations.
+        
+    ![](_images/webeditor/drag_collection.png)
+
+
+    Now, once you load collection, there are several parameters you need to specify, including the area of interest, the temporal extent, and the selection of bands. When you click on the collection, a window pops up, where you can define the necessary parameters for subsequent processing. 
```

Comment:
> When you click on the collection, a window pops up,

I think this should be clicking on any `load_collection` parameter, or clicking the "edit" icon in top right corner

Reply:
updated the sentence